src/base/memory_monitor_common.h

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2008 Search Solution Corporation
```

Comment:
```suggestion
 *
```
No need to add this license for newly created files